titus-api/src/test/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobModelSanitizationTest.java

```diff
@@ -233,6 +235,25 @@ public void testBatchJobWithIncompleteEfsDefinition() {
         assertThat(entitySanitizer.validate(sanitized)).isEmpty();
     }
 
+    @Test
+    public void testJobWithTooLargeEntryPoint() {
+        // Make key/value pair size 1MB
+        char[] manyChars = new char[1025];
+        Arrays.fill(manyChars, '0');
+        String bigString = new String(manyChars);
+
+        List<String> entryPoint = new ArrayList<>();
+        for (int i = 0; i < JobAssertions.MAX_ENTRY_POINT_SIZE_SIZE_KB; i++) {
+            entryPoint.add(bigString);
+        }
+
+        JobDescriptor<BatchJobExt> badJobDescriptor = oneTaskBatchJobDescriptor().but(jd -> jd.getContainer().toBuilder().withEntryPoint(entryPoint).build());
+
+        Set<ValidationError> violations = entitySanitizer.validate(badJobDescriptor);
+        assertThat(violations).hasSize(1);
+        assertThat(first(violations).getDescription()).contains("Entry point size exceeds the limit 16KB");
```

Comment:
add constant to the string so changing it doesn't mean you have to change the test?

Reply:
I followed the convention in this class.